### PR TITLE
Add distributed rate limiting scenario

### PR DESCRIPTION
**Pull request by RameshaKaru**
Add distributed rate limiting documentation and a scenario to demonstrate it

### CODE REVIEW

File: scenarios/scenario-12/swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    Pet:
+      content:
+        application/json:
+          schema:
+            "$ref": "#/components/schemas/Pet"
+        application/xml:
+          schema:
+            "$ref": "#/components/schemas/Pet"
+      description: Pet object that needs to be added to the store
+      required: true
```
Comment (dinusha92) [inline scenarios/scenario-12/swagger.yaml]: add a new line

File: scenarios/scenario-12/README.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+
+ ##### Deploying the artifacts
+
+1. **When distributed throttling is disabled**
+
+ - By default the distributed ratelimiting is disabled in controller-configs/controller_conf.yaml as below
+
+ ```
+ enabledGlobalTMEventPublishing: "false"
```
Comment (dinusha92) [inline scenarios/scenario-12/README.md]: should be aligned with other commands

File: scenarios/scenario-12/swagger.yaml (hunk 1 above)
Comment (RameshaKaru) [reply scenarios/scenario-12/swagger.yaml]: Added

File: scenarios/scenario-12/README.md (hunk 2 above)
Comment (RameshaKaru) [reply scenarios/scenario-12/README.md]: Alligned

**Pull request merged**